src/test/unit/function-info.test.js

```diff
@@ -30,3 +33,15 @@ describe('get-function-name', function() {
     expect(stripFunctionArguments('(root)')).toEqual('(root)');
   });
 });
+
+describe('remove-template-information', function() {
+  it('should remove template information', function() {
+    expect(
+      removeTemplateInformation('ns::Impl<void (ns::foo::*)(), (ns::bar)0>::fn')
+    ).toEqual('ns::Impl::fn');
+    expect(removeTemplateInformation('fn<ns::foo<ns::bar<a::b> > >')).toEqual(
+      'fn'
+    );
+    expect(removeTemplateInformation('foo/<bar')).toEqual('foo/<bar');
```

Comment:
I believe this check could go in a separate `it` test case, with a different name like "should not remove information we want to keep"